src/js/model/Recipe.js

```diff
@@ -83,6 +85,10 @@ class Recipe {
 			throw new Error("El ID del autor es inválido");
 		}
 	}
+	getRating() {
```

Comment:
Lamento decirte que esto no se puede hacer, porque es una dependencia circular. En java (creo) que sí se puede, pero node lleva este tema regular. Lo que está pasando es esto:

```js
// Recipe.js
const { getState } = require("../appState");

// appState.js
const Recipe = require("./model/Recipe");
```

Uno depende del otro, y el otro depende del uno. Y debido a cómo funciona node esto da problemas. Si quieres investigar más, [te dejo esta discusión](https://stackoverflow.com/questions/10869276/how-to-deal-with-cyclic-dependencies-in-node-js), pero por lo pronto lo más fácil es mover la función `getRating()` (que por cierto debería llamarse `getRatings` con s) fuera de `Recipe.js`.

Por ejemplo, a un archivo dentro de la carpeta `support` (que es una carpeta con funciones sueltas que son útiles) llamado `getRecipeRatings()`. En esa función le pasarías una receta y te devolvería sus ratings. La implementación sería la misma, remplazando `this` por el parámetro.